cucumber/authenticators_jwt/features/authn_jwt_token_schema.feature

```diff
@@ -448,4 +448,264 @@ Feature: JWT Authenticator - Token Schema
     cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
     """
 
+  Scenario: ONYX-10816 - Enforced Claims with Claims Mapping. Single enforced claim but not in token - 401 Error
+    Given I extend the policy with:
+    """
+    - !variable conjur/authn-jwt/raw/enforced-claims
+    - !variable conjur/authn-jwt/raw/mapping-claims
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/branch: valid
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I successfully set authn-jwt "enforced-claims" variable to value "ref"
+    And I successfully set authn-jwt "mapping-claims" variable to value "branch:ref"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00084E Claim 'ref (annotation: branch)' is missing from JWT token. Verify that you configured the host with permitted restrictions
+    """
+
+  Scenario: ONYX-10814 - Claim being mapped to another claim - 401 Error
+    Given I extend the policy with:
+    """
+    - !variable conjur/authn-jwt/raw/mapping-claims
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/sub: mysub
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I successfully set authn-jwt "mapping-claims" variable to value "sub:ref"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "sub":"mysub",
+      "ref":"mybranch"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00049E Resource restriction 'sub' does not match with the corresponding value in the request
+    """
+
+  Scenario: ONYX-10860 - Mapping claims configured but not populated - 401 Error
```

Comment:
In the original test we load to the policy the `enforced-claims` as well. I think its important since it may act different.